CMakeLists.txt

```diff
@@ -480,6 +480,11 @@ if (FLB_AWS_ERROR_REPORTER)
   FLB_DEFINITION(FLB_HAVE_AWS_ERROR_REPORTER)
 endif()
 
+# K8s token command is currently Linux-only.
```

Comment:
this definition is plugin-specific and must be inside plugins/filter_kubernetes/CMakeLists.txt

Reply:
Thanks so much